.travis.yml

```diff
@@ -1,16 +1,36 @@
 # A Travis CI configuration file.
 
+sudo: required
+
 language: go
 
 go:
   - 1.7
 
+services:
+  - docker
+
+git:
+  depth: 3
+
 install:
+  - export DEPLOY_BUILD_READY=false
   - go get -u github.com/golang/lint/golint
+  - go get -u github.com/stretchr/testify
 
 script:
   - make lint
   - make build
+  - source ./install_dependencies.sh
```

Comment:
do we need a test step to make sure the dependencies are ready? for example a curl command to test the openwhisk api is ready, or this step is not necessary?